api/proto/metrics/metrics.proto

```diff
@@ -123,12 +158,19 @@ message deviceMetric {
   repeated MetricItem metricItems = 8;
   uint64 runtimeStorageOverheadMB = 9;     // In MB
   uint64 appRunTimeStorageMB = 10;         // In MB
+
+  // systemServicesMemoryMB - OBSOLETE. Will no longer be used.
+  // Currently represents whole usage for Dom0
```

Comment:
This line is incorrect (there is no dom0 with KVM for instance). So don't add it.

Reply:
Done. Removed it.